zad1/main.rb

```diff
@@ -1,13 +1,26 @@
 def sort_letters(string)
-
+	puts string.chars.sort.join
```

Comment:
Metoda rzeczywiście sortuje alfabetycznie znaki w `string`, ale zwracana wartość to `nil`, czyli robi coś innego niż sugerowałaby nazwa - teraz mogłaby nazywać się `print_sorted_letters` 🤔
```ruby
sort_letters('tetriandoch') # => nil 😢 
```

Reply:
podobnie inne metody z tego zadania, zwracając coś innego niż `nil` stałyby się dużo bardziej użyteczne 🐸